src/controllers/template-controller.ts

```diff
@@ -1,21 +1,28 @@
 import { join } from "path";
-import { extensionPath } from '../extension';
 import { postError, showStatusMessage } from '../helper/common';
 import { templateRepo } from '../helper/user-settings';
+import { homedir } from 'os';
+import { copyFileSync, existsSync, mkdirSync } from "fs";
 
 export let localTemplateRepoPath: string;
 let templateZip: string;
 
 export async function downloadTemplateZip() {
   const download = require('download');
   const tmp = require('tmp');
+  const docsAuthoringHomeDirectory = join(homedir(), 'Docs Authoring');
+  const offlineZip = join(docsAuthoringHomeDirectory, 'learn-scaffolding-main.zip');
+  if (!existsSync(docsAuthoringHomeDirectory)) { 
+    mkdirSync(docsAuthoringHomeDirectory);
+  }
   localTemplateRepoPath = tmp.dirSync({ unsafeCleanup: true }).name;
   showStatusMessage(`Temp working directory ${localTemplateRepoPath} has been created.`);
   try {
     await download(templateRepo, localTemplateRepoPath);
     templateZip = join(localTemplateRepoPath, 'learn-scaffolding-main.zip');
+    copyFileSync(templateZip, offlineZip);
   } catch (error) {
-    templateZip = join(extensionPath, 'offline-template-zip', 'learn-scaffolding-main.zip');
+    templateZip = offlineZip;
```

Comment:
I think we need to check if `learn-scaffolding-main.zip` exists in `docsAuthoringHomeDirectory`. if not, should fallback to the zip in `offline-template-zip`

Reply:
Hey @lamebrain , pushed an update for this one. Thx